src/app/services/baw-api/harvest/harvest.service.ts

```diff
@@ -104,6 +104,17 @@ export class HarvestsService implements StandardApi<Harvest, [IdOr<Project>]> {
   ): Observable<Harvest> {
     return this.shallowHarvestsApi.updateMappings(model, mappings);
   }
+
+  public isAbortable(
+    model: Harvest
+  ): boolean {
+    const transitionableStates: HarvestStatus[] = [
+      "metadataReview",
+      "uploading"
+    ];
+
+    return transitionableStates.includes(model.status);
+  }
```

Comment:
this should be in the harvest model, not the service

Reply:
Done